cmk/special_agents/utils/prometheus.py

```diff
@@ -43,10 +43,13 @@ def extract_connection_args(config):
 
     address = config["host_address"] if connect_type == "ip_address" else config["host_name"]
 
-    if "path-prefix" in connect_settings:
-        address = f"{connect_settings['path-prefix']}{address}"
+    path_prefix = connect_settings.get("path-prefix", "")
+    if path_prefix:
+        path_prefix = path_prefix + "/"
```

Comment:
Ok to ignore: This is reimplementing existing functionality inside of parse_api_url. I would change it.